element-hq/element-x-android
# Add support for sending images from the keyboard

## Type of change


## Content


Add ability to send media from the keyboard

Need https://github.com/matrix-org/matrix-rich-text-editor/pull/915

## Motivation and context


Better integration with the major Android keyboards, and parity with EA.
Closes #1977 

## Screenshots / GIFs


![SendImageFromKb](https://github.com/element-hq/element-x-android/assets/3940906/8e74092b-5c9d-4f67-8883-91bb1b8ce47b)

## Tests



- Open a room with permission to send message
- From the keyboard, select an image
- observe that the image is sent to the room

## Tested devices

- OS version(s):

## Checklist
